tests/unit/proxy/test_elasticsearch.py

```diff
@@ -93,7 +95,8 @@ def setUp(self) -> None:
                                              column_names=['test_col1', 'test_col2'],
                                              tags=self.mock_empty_tag,
                                              badges=self.mock_empty_badge,
-                                             last_updated_timestamp=1527283287)
+                                             last_updated_timestamp=1527283287,
+                                             programmatic_descriptions=[])
 
         self.mock_result2 = MockSearchResult(name='test_table2',
```

Comment:
hey @jinhyukchang here is a mock result where programmatic_descriptions aren't specified